game/templates/game/landing.html

```
@@ -1,1992 +1,1420 @@
-    <section class="platform-stats-section">
-      <div class="stats-container">
-        <div class="stat-box">
-          <div class="stat-icon">⚔️</div>
-          <div class="stat-number-wrapper">
-            <span class="stat-num" data-target="1250000">0</span><span class="stat-plus">+</span>
+      <!-- Game Timer -->
+      <div class="bento-card">
+        <div class="bento-visual timer-visual">
+          <div class="digital-clock">05<span class="blink">:</span>00</div>
+          <div class="timer-controls">
+            <button class="t-btn">⏸</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)